<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLivro</class>
 <widget class="QDialog" name="TelaLivro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="inputNome">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TelaLivro</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNome">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nome Livro: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="addLivro">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputNome</sender>
   <signal>textChanged(QString)</signal>
   <receiver>inputNome</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
